modules/resources/deployment-script-import-acr/main.bicep

```diff
@@ -0,0 +1,105 @@
+@description('The name of the Azure Container Registry')
+param acrName string
+
+@description('The location to deploy the resources to')
+param location string = resourceGroup().location
+
+@description('How the deployment script should be forced to execute')
+param forceUpdateTag  string = utcNow()
+
+@description('Version of the Azure CLI to use')
+param azCliVersion string = '2.30.0'
+
+@description('Deployment Script timeout')
+param timeout string = 'PT30M'
+
+@description('The retention period for the deployment script')
+param retention string = 'P1D'
+
+@description('An array of Azure RoleIds that are required for the DeploymentScript resource')
+param rbacRolesNeeded array = [
+  'b24988ac-6180-42a0-ab88-20f7382dd24c' //Contributor is needed to import ACR
+]
+
+@description('Name of the Managed Identity resource')
+param managedIdName string = 'id-ContainerRegistryImport'
+
+@description('An array of fully qualified images names to import')
+param images array = [
```

Comment:
I don't think this parameter should have a default value. Anyone using this script would presumably have a very specific list of images they want to reference. I'd be worried users will accidentally get this default value when it is not their intention.

Reply:
Agreed. 👍